.github/workflows/ci.yaml

```
@@ -8,59 +8,60 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
+  # Enable sccache to speed up builds.
```

```suggestion
  # This enables sccache to speed up builds.
```
To make it clear this isn't telling the user they need to _do something_ to enable sccache.